### DISCUSSION TITLE
How to manipulate richtexteditor Html ?

### DISCUSSION THREAD

**Discussion by erenergul** [filamentphp/filament#2165] [Help]
Hello , I have a specific class for 

```
<ul class="pref" >
<li>asdasddsa</li>
</ul>
```

How can i see source code ? and change it ? I couldn't find any way. I tried to use external package tinymce from plugin page. Still can not see any source code. Is there any way to change it ?

**Comment by danharrin** [comment]
It's not possible with Trix, which we use for our default Rich Editor

**Reply by erenergul** [reply]
Thank you for your answer. I use backend foreach  loop instead of  text editor. Hope somebody create new plugin about it.

**Reply by danharrin** [reply]
You could use a `str_replace()` when you render the content
